<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MathGame</class>
 <widget class="QWidget" name="MathGame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>410</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="answerButtonA">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>480</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(210, 210, 210);
text-align: left;
padding-left: 80px;</string>
   </property>
   <property name="text">
    <string>Button A</string>
   </property>
  </widget>
  <widget class="QPushButton" name="answerButtonB">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>480</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(210, 210, 210);
text-align: left;
padding-left: 80px;</string>
   </property>
   <property name="text">
    <string>Button B</string>
   </property>
  </widget>
  <widget class="QPushButton" name="answerButtonC">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>480</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(210, 210, 210);
text-align: left;
padding-left: 80px;</string>
   </property>
   <property name="text">
    <string>Button C</string>
   </property>
  </widget>
  <widget class="QPushButton" name="answerButtonD">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>330</y>
     <width>480</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(210, 210, 210);
text-align: left;
padding-left: 80px;</string>
   </property>
   <property name="text">
    <string>Button D</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelA">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>60</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>25</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(85, 87, 83);
color: #eee;
</string>
   </property>
   <property name="text">
    <string>A</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelB">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>60</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>25</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(85, 87, 83);
color: #eee;
</string>
   </property>
   <property name="text">
    <string>B</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelC">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>60</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>25</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(85, 87, 83);
color: #eee;
</string>
   </property>
   <property name="text">
    <string>C</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelD">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>330</y>
     <width>60</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>25</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(85, 87, 83);
color: #eee;
</string>
   </property>
   <property name="text">
    <string>D</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="questionLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>480</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">
background-color: rgb(210, 210, 210);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
